### PR TITLE
Add SME notes to netstat article

### PR DESCRIPTION
**Pull request by DonNateR**
- Fixes issue #913 (SME review of article)

### CODE REVIEW

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-efd283bc89/how-to/)

**Comment by kmsholcomb** [comment]
Hi @DonNateR. My edits are in the attached Word and PDF docs. They are fairly minimal, but please let me know if you have any questions. Thanks!

[Check Listening Ports with netstat_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/364266/Check.Listening.Ports.with.netstat_editsKH.pdf)
[Check Listening Ports with netstat_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/364267/Check.Listening.Ports.with.netstat_editsKH.docx)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-67680d7369/how-to/). :bow:

File: content/cloud-servers/checking-listening-ports-with-netstat.md [hunk 1]
```diff
@@ -1,37 +1,39 @@
...
-last_modified_by: Stephanie Fillmon
+last_modified_date: '2016-07-15'
+last_modified_by: Nate Archer
 product: Cloud Servers
 product_url: cloud-servers
 ---
 
 If you're troubleshooting a service that you know [is running normally](/how-to/checking-system-load-on-linux)
-the next step is to make sure it's listening to the right network port.
+the next step is to ensure sure it's listening on the correct network port.
```
Comment (stephamon) [inline content/cloud-servers/checking-listening-ports-with-netstat.md]: Double "sure"s here. Should be "the next step is to ensure that it's listening..."

File: content/cloud-servers/checking-listening-ports-with-netstat.md [hunk 2]
```diff
@@ -44,37 +46,34 @@ that are listening on three different sockets.
...
     tcp        0      0 :::80                       :::*                        LISTEN      8448/httpd
 
 ### Analyze the results
 
-Common outcomes are:
+Common outcomes are as follows:
 
 -   Nothing is listening on the port. Check the service configuration
-    files then try restarting the service.
+    files, and then try restart the service.
```
Comment (stephamon) [inline content/cloud-servers/checking-listening-ports-with-netstat.md]: ...and then try **to** restart the service.

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-d2b8b4a1bc/how-to/). :bow:

**Comment by stephamon** [comment]
👍

**Pull request merged**